core/domain/user_jobs_one_off.py

```diff
@@ -407,6 +407,33 @@ def reduce(key, values):
         yield (key, len(values))
 
 
+class AddTopicIdsAndStoryIdsOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that adds story_ids and topic_ids in CompletedActivitiesModel.
+
+    NOTE TO DEVELOPERS: This job can be deleted after it is run in April 2021
+    release.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [
+            user_models.CompletedActivitiesModel,
+            user_models.IncompleteActivitiesModel
+        ]
+
+    @staticmethod
+    def map(model_instance):
+        model_instance.update_timestamps(
+            update_last_updated_time=False)
+        model_instance.put()
+        yield ('SUCCESS', model_instance)
```

Comment:
As discussed with @seanlip, the project Redesigning and Updating the learner dashboard page requires this migration job to run on the server before making any backend or frontend changes during the GSoC phase. He suggested this because we have the Python migration project going on, which means that there'll be 1-2 months in which no jobs can be run at all due to the switching out of MapReduce infrastructure with Apache Beam. So, migrate jobs in as early as possible once committed to them.
This job adds the topic_ids and story_ids field inside the CompletedActivitiesModel and IncompleteActivitiesModel which will be used to store the completed and incomplete stories and topics on the server and display them on the learner dashboard page.

Reply:
But since you're adding basically empty repeated fields there is no need for a job. You can test this locally if you want. Run local server with old code and have some activity models created. Then keep the datastore (there is a command for that in the start script) and start a server with your added changes and try to add some values in the story or topic IDs fields of the previously created models.

Reply:
Yeah, it works! So just to clarify, I only need to add the fields inside the Model and domain object and since it's an empty repeated field, it does not require any one-off job, right?

Reply:
@krishita30j I don't think your message included any of the implementation details (sorry if I missed it). I agree with @vojtechjelinek that addition of field properties do not need jobs. It's only when you migrate JsonProperties that have a schema version.

Reply:
@krishita30j Yes, exactly.

Reply:
@seanlip @vojtechjelinek thanks! I'll keep this in mind for the future. I'll close this PR then.